README.md

```diff
@@ -77,6 +77,17 @@ Generated events with end-to-end tests or with manual navigation on the LMS serv
 
 3. You should see appearing a list of all the messages you have generated!
 
+## Keycloak
+
+The keycloak SSO service is pre-configured for the `fun-mooc` realm. It can be
+accessed at [http://localhost:8080](http://localhost:8080). Administrator
+credentials are: `admin:pass`.
```

Comment:
I would add an additional sentence to explicitly write that it does not need more than a `make run` or `make bootstrap` on the playground to launch the service.